### DISCUSSION TITLE
New interface for navigating code expects a monitor that’s too wide

### DISCUSSION THREAD

**Discussion by acli** [community/community#29324] [Other Feature Feedback, Questions, & Ideas] [answered]
The left side of the current interface for code listings is reserved for navigating the source tree. There doesn’t seem to be any way to turn this off or hide this. On a 1024px wide monitor this takes up almost half of the screen, which makes it very difficult to actually read the source code.

A 1024px wide monitor isn’t uncommon; it’s just a 1920px wide monitor in portrait mode.

**Comment by colinwm** [comment, marked as answer]
You can hide the code view, by clicking on the chevron:

![image](https://user-images.githubusercontent.com/78169714/184906934-6b8c90cf-20f7-42dc-aeac-b328be75a2b0.png)
